lib/globals.d.ts

```diff
@@ -21,3 +21,49 @@ declare module '*.json' {
   const value: Record<string, any>;
   export = value;
 }
+
+// these  declartions will be removed once the types.ts file for @renovate/pep440 is ready
```

Comment:
```suggestion
// these declarations will be removed once the types.ts file for @renovate/pep440 is ready
```

If you're going to keep this comment, here's a fix for the grammar.

If you're going to remove the comment you can ignore this suggestion.